### PR TITLE
`CreateChainTx` specification

### PR DESCRIPTION
**Pull request by cgcardona**
First draft of `CreateChainTx` specification including:

* Contents
* Gantt
* Proto
* Example

### CODE REVIEW

**Review by yulin-dong — DISMISSED** [review]

File: docs/build/references/platform-transaction-serialization.md [hunk 1]
```diff
@@ -891,6 +891,196 @@ Let’s make an unsigned add delegator tx that uses the inputs and outputs from
...
+  0xf4, 0xea, 0xb4, 0x8f, 0x0d, 0xa8, 0x24, 0x6b, 
+  0x62, 0xfb, 0x29, 0xa0, 0x14, 0x1b, 0x65, 0x41, 
+  0x99, 0x17, 0x64, 0x40, 0x83, 0x2e, 0x2b, 0xc8, 
+  0xed, 0x01, 0x10, 0x25, 0x47, 0x7d, 0x3d, 0xa7, 
+  0x17, 0xb5, 0xe6, 0xe1, 0xe7, 0xf2, 0x2f, 0x74, 
+  0xb4, 0x39, 0x02, 0x91, 0xe9, 0x3f, 0x93, 0x76, 
+  0xf8, 0x60, 0xa1, 0x2c, 0x15, 0xc9, 0xf1, 0x02, 
+  0x91, 0xc3, 0x4e, 0x27, 0x85, 0x7c, 0x6d, 0x20, 
+  0x01, 
+]
```
Comment (danlaine) [inline docs/build/references/platform-transaction-serialization.md:1081]: Should we include the signatures here? This is supposed to be an unsigned tx example, no?

Comment (cgcardona) [reply docs/build/references/platform-transaction-serialization.md]: Good catch. Removed.

File: docs/build/references/platform-transaction-serialization.md [hunk 2]
```diff
@@ -891,6 +891,163 @@ Let’s make an unsigned add delegator tx that uses the inputs and outputs from
...
+
+### **Unsigned Create Chain Tx Example**
+
+Let’s make an unsigned create chain tx that uses the inputs and outputs from the previous examples:
+
+* **`BaseTx`**: `"Example BaseTx as defined above with ID set to 0f"`
+* **`SubnetID`**: `24tZhrm8j8GCJRE9PomW8FaeqbgGS4UAQjJnqqn8pq5NwYSYV1`
+* **`ChainName`**: `EPIC AVM`
+* **`VMID`**: `avm`
+* **`FxIDs`**: `0x736563703235366b316678000000000000000000000000000000000000000000`
```
Comment (StephenButtolph) [inline docs/build/references/platform-transaction-serialization.md:953]: I feel like `VMID` and `FxIDs` should either both be human readable here - or both hex. Also `FxIDs` should be an array right?

Comment (cgcardona) [reply docs/build/references/platform-transaction-serialization.md]: Good call. Resolved.

**Pull request merged**